app/models/rule.rb

```diff
@@ -8,11 +8,13 @@ class Rule < ActiveRecord::Base
     points_for_win:       0,
     points_for_loss:      1,
     points_for_tie:       2,
-    points_for_time_win:  3
+    points_for_time_win:  3,
```

Comment:
Tools that hound me about stuff that I don't care about makes me :crying_cat_face: 

Burn that setting in your Hound configuration. Do it for me Johno :smiley: